client/client_test.go

```diff
@@ -160,14 +154,6 @@ func TestAddLeaf(t *testing.T) {
 		},
 	} {
 		client := New(logID, test.client, testonly.Hasher, env.PublicKey)
-		{
-			ctx, cancel := context.WithDeadline(ctx, time.Now().Add(timeout))
-			defer cancel()
-			if err, want := client.AddLeaf(ctx, []byte(test.desc)), codes.DeadlineExceeded; grpc.Code(err) != want {
-				t.Errorf("AddLeaf(%v): %v, want, %v", test.desc, err, want)
-				continue
-			}
-		}
```

Comment:
I'm a bit uncertain about this change as I don't understand what the point of the code I removed is (Chesterton's Fence)!  @gdbelvin, what was this supposed to do?